lang/src/org/partiql/lang/compiler/PartiQLCompilerBuilder.kt

```diff
@@ -51,6 +53,7 @@ class PartiQLCompilerBuilder private constructor() {
     private var customFunctions: List<ExprFunction> = emptyList()
     private var customProcedures: List<StoredProcedure> = emptyList()
     private var customOperatorFactories: List<RelationalOperatorFactory> = emptyList()
+    private var plugin: Plugin = LegacyPlugin()
```

Comment:
Why is it called LegacyPlugin? Also, we shouldn't have just one plugin

Reply:
Currently, it still contains legacy types, functions & operators. Later when we start to work on function overloading, and finish the work on rewriting each function/operator specific to its arguments, we can call it `StandardPlugin`.